src/Debug/ProfilerSection.php

```diff
@@ -51,6 +51,12 @@ final class ProfilerSection
 
     private int $end;
 
+    /**
+     * @var array Array of start and end times of paises which will be removed from the final duration.
+     * Index 0 will be the start time and index 1 will be the end time.
```

Comment:
It would probably easier to read if pauses were stored in following format: `['start' => int, 'end' => int|null]`.